### DISCUSSION TITLE
why did this repo repurpose the "golang-cross" repo?

### DISCUSSION THREAD

**Discussion by seeekr** [vercel/turborepo#2344] [Help] [answered]
I found it a little weird that, after hearing of this new tool "turbo" and opening its GH repo, it showed me that I had already starred it previously... And with a little digging into the commit history the reason becomes apparent:

https://github.com/vercel/turbo/tree/131c98855224978972e64dca5896956de775eacb

This repo used to be "golang-cross" (Docker image for cross-compiling go code), which I had indeed starred a long time ago.

Can anyone explain the rationale? Or is GH just drunk here somehow?

Thanks! And of course appreciate the work going into this new tool and that it's open source!

**Comment by nathanhammond** [comment, marked as answer]
Nah, we merged those commits in separately. This repo was previously `vercel/turborepo` which you presumably starred.
